File: app/src/main/java/org/mhsgv/portal/form/Form2Fragment.java
package org.mhsgv.portal.form;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import org.mhsgv.portal.R;

public class Form2Fragment extends FormFragment {

    private static String TAG = Form2Fragment.class.getSimpleName();


    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        View view = super.createView(inflater, container, savedInstanceState, R.layout.fragment_form2);
        // Form specific implementation starts here.
        return view;
    }
}

File: app/src/main/java/org/mhsgv/portal/form/Form2Fragment.kt
package org.mhsgv.portal.form

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import org.mhsgv.portal.R

class Form2Fragment : FormFragment() {
    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {
        // Form specific implementation starts here.
        return super.createView(inflater, container, savedInstanceState, R.layout.fragment_form2)
    }

    companion object {
        private val TAG = Form2Fragment::class.java.simpleName
    }
}